### ISSUE TITLE
AdultDvDEmpire and HotMovies scrapers failing on Network Error: Could not load the requested resource

### ISSUE THREAD

**Issue by Shades6666** [Komet/MediaElch#1811]
**Scrapers that don't work:**

 - [ ] IMDb
 - [ ] TMDb
 - [ ] TmdbConcert
 - [ ] TvTunes
 - [ ] AEBN
 - [x] AdultDvDEmpire
 - [x] HotMovies
 - [ ] VideoBuster

**Details which are wrong / not loaded:**
Both AdultDvDEmpire and HotMovies fail with "Network Error: Could not load the requested resource".  All other scrapers working correctly.
Checking the logs, both appear to be getting stuck on age verification pages:

[AdultDvdEmpireApi] Network Error: "Insecure redirect" for URL QUrl("http://www.adultdvdempire.com:80/AgeConfirmation?url2=/allsearch/search?view=list&q=Searched Movie")

[HotMoviesApi] Network Error: "Insecure redirect" for URL QUrl("http://www.hotmovies.com:80/AgeConfirmation?url2=/adult-movies/search?q=Searched Movie")

**MediaElch Version:**
 - [x] 2.12.1-dev (nightly)
 - [x] 2.12.0 (stable)
 - [x] 2.10.7-dev (nightly) -- I also tried an old nightly that I still had and it failed with the same error.

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [x] Linux (distribution: Ubuntu 24.04.1 LTS (noble) using appImage)

**Additional context:**

**Comment by bugwelle** [comment]
Hi,

thanks for reporting! Sorry for the delay. See https://github.com/Komet/MediaElch/issues/1710 for details.

I can confirm the issue. I also found the root cause and am working on a fix.

Regards,
Andre

**Comment by bugwelle** [comment]
This may take a bit longer: The pages now have an age restriction. Settings a cookie alone does not seem to work properly.

----

Notes:

```js

function ageConfirmationClicked() {
	$.ajax({
		url: "Account/AgeConfirmation",
		data: ({
			ageConfirmationClicked: true
		}),
		success: function () {
					
			window.location = $(ageConfirmationButton).attr('data-url');
					   
		}
	});
}
```

**Comment by bugwelle** [comment]
Thanks again for reporting. I've fixed this in #1816. The next Nightly version should work. :)
